docs/snippets/license_preamble.txt

```diff
@@ -0,0 +1,20 @@
+This file is part of %package_name%.
+
+Developed for the LSST Data Management System.
+This product includes software developed by the LSST Project
+(http://www.lsst.org).
+See the COPYRIGHT file at the top-level directory of this distribution
+for details of code ownership.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
I've removed the LSST link from here because it is confusingly different to an actual GPL license and includes every license that LSST has found in code it uses. An option is to add the link to the root of the LSST licenses directory to the top of the license text associated with the lsst.org link.

Reply:
This is a pattern I see in many open source projects: https://github.com/astropy/astropy/tree/master/licenses But I can't personally comment on what we should actually do here. Having the real GPLv3 license as our own license text seems like a good idea, though.

Reply:
That pattern seems fine  of putting all the myriad sublicenses in a subdirectory but having the proper license at the top level. The LSST "license" combined them all into a single confusing file which Github then has no idea what to do with. My other comment from that link is that I had no idea we have an AURA license.